### DISCUSSION TITLE
Any guidelines for totalseqc? (137 antibodies)

### DISCUSSION THREAD

**Discussion by oshiomah1** [campbio/decontX#34] [General]
I have a citeseq panel with 137 antibodies, I have no idea what values to use for the delta and background sd. The tutorial says "We tested the default values on datasets ranging 5k to 10k droplets and 10 to 30 ADTs and the results are reasonable."

```

out <- decontPro(counts,
                 clusters,
                 delta_sd = 2e-4,
                 background_sd = 2e-5)
```

**Comment by joshua-d-campbell** [comment]
Hi @oshiomah1, we have used the current default priors successfully with Total-seq data (delta_sd = 2e-5 and background_sd =. 2e-6). We have also used delta SD = 2e-4 and a background SD = 2e-5 successfully with a previous Total-seq dataset. Let us know what works best for your data! I'm going to move this to a Discussion as it may be useful for other people.
